### PR TITLE
cleanup: remove redundant testing, add logs

### PR DESCRIPTION
**Pull request by joschisan**

### CODE REVIEW

**Review by dpc — COMMENTED** [review]
LGTM, but @m1sterc001guy should check.

File: modules/fedimint-lnv2-tests/src/bin/tests.rs [hunk 1]
```diff
@@ -142,72 +207,88 @@ async fn test_self_payment_success(dev_fed: &DevJitFed) -> anyhow::Result<()> {
...
+        .await?;
+
     let gw_lnd = dev_fed.gw_lnd().await?;
     let gw_cln = dev_fed.gw_cln().await?;
+
     let lnd = dev_fed.lnd().await?;
     let cln = dev_fed.cln().await?;
 
-    // Verify HOLD invoices still work, create one now for payment later
+    info!("Verify HOLD invoices still work, create one now for payment later");
```
Comment (tvolk131) [inline modules/fedimint-lnv2-tests/src/bin/tests.rs:239]: Nit: The other logs here are all present tense ("Testing"), can we change this to "Verifying HOLD invoices..."

**Review by tvolk131 — APPROVED** [review]
LGTM, just one nit

File: modules/fedimint-lnv2-tests/src/bin/tests.rs (hunk 1 above)
Comment (tvolk131) [reply modules/fedimint-lnv2-tests/src/bin/tests.rs:239]: Meh, actually I think it's fine the way it is

**Pull request merged**